schema/models_directory/configuration_interaction.json

```diff
@@ -0,0 +1,63 @@
+{
+    "schemaId": "models-directory-configuration-interaction",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "configuration interaction schema",
+    "description": "configuration interaction wavefunction model",
+    "allOf": [
+        {
+            "$ref": "../model.json"
+        }
+    ],
+    "properties": {
+        "type": {
+            "enum": [
```

Comment:
We should probably retain the structure with having a human-readable `name` and a unique computer-friendly `slug`, maybe having something like:

```
type: {
   name: "Configurational Interaction, Singles",
   slug: "cis"
}
```